src/query/xasl.h

```diff
@@ -923,6 +924,15 @@ struct xasl_stat
   UINT64 fetch_time;
 };
 
+struct sq_stat
+{
+  UINT64 sq_cache_hit;
+  UINT64 sq_cache_miss;
+  UINT64 sq_cache_size;
+  UINT64 sq_cache_size_max;
+  UINT64 sq_cache_miss_max;
```

Comment:
These three are not related to statistical information.
How about this?
```c++
struct sq_cache
{
  MHT_TABLE *ht;
  UINT64 max_size; /* UINT64 ? */
  UINT64 size

  int flag; /* I wonder if it is necessary. */

  struct
    {
    UINT64 hit;
    UINT64 miss;
  } stats;

  UINT64 max_miss; /* I wonder if it is necessary. */
};
```

Reply:
This is an example.
`sq_cache` must be created as a separate struct and typedefed.